.github/workflows/go.yml

```diff
@@ -36,5 +36,5 @@ jobs:
     - name: Run staticcheck
       run: |
         cd yb-voyager
-        go install honnef.co/go/tools/cmd/staticcheck@2023.1.7
+        go install honnef.co/go/tools/cmd/staticcheck@2024.1.1
```

Comment:
Mention in the description that the updated version of this and reason.

Reply:
Done